models/timm/efficientnet_b1.py

```diff
@@ -0,0 +1,19 @@
+# labels: name::efficientnet_b1 author::timm
```

Comment:
Is this going to have a name collision with `models/torch_hub/efficientnet_b1.py`? What happens in the report if I benchmark both `torch_hub` and `timm` corpus?

Reply:
Great catch. You are right that we now have some overlaps. More precisely, the following models are overlapping:

```
torch_hub/resnet34.py
torch_hub/vgg19_bn.py
torch_hub/densenet169.py
torch_hub/resnext50_32x4d.py
torch_hub/efficientnet_b3.py
torch_hub/efficientnet_b4.py
torch_hub/wide_resnet101_2.py
torch_hub/inception_v3.py
torch_hub/vgg19.py
torch_hub/densenet121.py
torch_hub/densenet161.py
torch_hub/convnext_large.py
torch_hub/vgg11.py
torch_hub/vgg13.py
torch_hub/efficientnet_b0.py
torch_hub/convnext_tiny.py
torch_hub/efficientnet_b2.py
torch_hub/efficientnet_b5.py
torch_hub/vgg16.py
torch_hub/resnet101.py
torch_hub/resnet152.py
torch_hub/efficientnet_b1.py
torch_hub/convnext_small.py
torch_hub/resnet18.py
torch_hub/efficientnet_b6.py
torch_hub/efficientnet_b7.py
torch_hub/convnext_base.py
torch_hub/vgg13_bn.py
torch_hub/vgg11_bn.py
torch_hub/vgg16_bn.py
torch_hub/resnet50.py
torch_hub/wide_resnet50_2.py
torch_hub/densenet201.py
torch_hub/resnext101_32x8d.py
```

What would be your recommendation here?

Reply:
`modelname_library` is an option. (At the risk of verbose file names)

Reply:
For all the models in all corpora or just the duplicates?

Reply:
We could leave the script file names alone but change the build names to be like reset34_timms_hash

Reply:
Yeah thats a better idea. Just edit the build names to include the library